### PR TITLE
fix 1064: submit forms not just on submit button click

### PR DESCRIPTION
**Pull request by alexander-elgin**
Part of oasisprotocol/oasis-wallet-web#1064

### CODE REVIEW

**Review by pro-wh — APPROVED** [review]
Implemented as requested, thanks!

File: src/app/components/MnemonicValidation/index.tsx [hunk 1]
```diff
@@ -80,7 +80,6 @@ export function MnemonicValidation(props: Props) {
               type="submit"
               label={t('openWallet.mnemonic.import', 'Import my wallet')}
               primary
-              onClick={onSubmit}
```
Comment (pro-wh) [inline src/app/components/MnemonicValidation/index.tsx:83]: the linter wants us to collapse this into one line after the removal. @alexander-elgin would you be able to make that change for us too?

Comment (alexander-elgin) [reply src/app/components/MnemonicValidation/index.tsx:83]: @pro-wh Done. Sorry, I did not check it at the first time

**Comment by pro-wh** [comment]
thanks, merging

**Pull request merged**